### DISCUSSION TITLE
Virtual Members

### DISCUSSION THREAD

**Discussion by wjohn007** [tasmota/Berry_playground#26] [Tutorials and Code Blocks]
Please find my fork 
https://github.com/wjohn007/Berry_playground/tree/main/tutorial-code-blocks/virtualMembers

It deals with 'virtual members'.

John

**Comment by dkebler** [comment]
Thanks for your contribution.  Looks good.  

Add a link in your readme to this discussion and make a PR and I'll merge it.

`[virtual members discussion](https://github.com/tasmota/Berry_playground/discussions/26)`

**Comment by wjohn007** [comment]
PR Done
#27
